libvast/src/experimental_table_slice_builder.cpp

```diff
@@ -24,6 +27,9 @@
 #include <arrow/io/api.h>
 #include <arrow/ipc/writer.h>
 
+#include <ranges>
```

Comment:
This looks like an IWYU leftover.

I recommend disabling it for clangd using `-header-insertion=never`.